lib/ansible/plugins/doc_fragments/hwc.py

```diff
@@ -0,0 +1,78 @@
+# Copyright: (c) 2018, Huawei Inc.
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+
+    # HWC doc fragment.
+    DOCUMENTATION = '''
+options:
+    identity_endpoint:
+        description:
+            - The Identity authentication URL.
+        type: str
+        required: true
+    user_name:
```

Comment:
I'd drop `_name`

Reply:
If I drop "_name", this will be "user". This configuration("user") means that support both the user name and the user id. But we only support user name.

Reply:
You could add a comment in the description that currently only usernames are supported, and not user IDs. Then you can add support for user IDs in the future without having to introduce a new option.

Reply:
Good idea!  Done